.github/workflows/check-edit-links.yml

```diff
@@ -0,0 +1,49 @@
+name: Weekly Link Checker
+
+on:
+  schedule:
+    - cron: '0 0 * * 0' # Runs every week at midnight on Sunday
+  workflow_dispatch:
+
+jobs:
+  check-links:
+    name: Run Link Checker and Notify Slack
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
```

Comment:
Use the default node version given in `.nvmrc` file.

Reply:
used `node-version-file` for `.nvmrc` version.